app/Console/Commands/SendAcceptanceReminder.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Asset;
+use App\Models\CheckoutAcceptance;
+use App\Models\Setting;
+use App\Models\User;
+use App\Notifications\CheckoutAssetNotification;
+use App\Notifications\CurrentInventory;
+use Illuminate\Console\Command;
+use Illuminate\Support\Facades\Notification;
+
+class SendAcceptanceReminder extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'snipeit:acceptance-reminder';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'This will send users with unaccepted assets a reminder to accept them.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $acceptances = CheckoutAcceptance::pending()->where('checkoutable_type', 'App\Models\Asset')->with(['assignedTo', 'checkoutable.assignedTo', 'checkoutable.model', 'checkoutable.adminuser'])->get();
+
+        $count = 0;
+        $unacceptedAssets = $acceptances
+            ->filter(function($acceptance) {
+                return $acceptance->checkoutable_type == 'App\Models\Asset';
+            })
+            ->map(function($acceptance) {
+                return ['assetItem' => $acceptance->checkoutable, 'acceptance' => $acceptance];
+            });
+
+        foreach($unacceptedAssets as $unacceptedAsset) {
+            if ($unacceptedAsset['acceptance']->assignedTo) {
+
+                if (!$unacceptedAsset['acceptance']->assignedTo->locale) {
+                    Notification::locale(Setting::getSettings()->locale)->send(
+                        $unacceptedAsset['acceptance']->assignedTo,
+                        new CheckoutAssetNotification($unacceptedAsset['assetItem'], $unacceptedAsset['acceptance']->assignedTo, $unacceptedAsset['assetItem']->adminuser, $unacceptedAsset['acceptance'], '')
+                    );
+                } else {
+                    Notification::send(
+                        $unacceptedAsset['acceptance']->assignedTo,
+                        new CheckoutAssetNotification($unacceptedAsset['assetItem'], $unacceptedAsset['acceptance']->assignedTo, $unacceptedAsset['assetItem']->adminuser, $unacceptedAsset['acceptance'], '')
+                    );
+                }
+                $count++;
+            }
+        }
+
+        if ($unacceptedAsset['acceptance']->assignedTo->email == ''){
+            return "no email";
```

Comment:
A return here seems like it might shortcut the rest when the first user who didn't accept and also doesn't have an email address happens. Can we maybe instead assume no one has an email address, stuff the ones that do into a `$count++` for the summary at the end?